Clarify when RCA should be ready to be shared publicly

Tessg22 opened the pull request:
Follow up on Incident Management Process improvement for RCA that has been delayed in public sharing due to unclear responsibilities of the Incident Owner. https://issues.redhat.com/browse/OHSS-15947

sesheta commented:
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **hemajv** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @hemajv` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=operate-first%2FSRE).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/operate-first/SRE/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>
<!-- META={"approvers":["hemajv"]} -->

anishasthana reviewed (approved):
/lgtm

sesheta commented:
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/operate-first/SRE/pull/53#pullrequestreview-580750754" title="LGTM">anishasthana</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=operate-first%2FSRE).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/operate-first/SRE/blob/master/OWNERS)~~ [anishasthana]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>
<!-- META={"approvers":[]} -->

HumairAK commented on README.md:
```diff
@@ -15,21 +15,27 @@ The components we currently have available are:
+For all the services being monitored, we define runbooks to provide information on how to administer, debug and effectively troubleshoot common problems of a service.
 
-TO DO
+The runbook for each of the individual services are defined in the [`runbooks`](https://github.com/operate-first/SRE/tree/master/runbooks)
```
Can we change the wording here to sound more like we're aiming, or  working towards adding runbooks for monitored services, and we'll add more as time goes on. Atm it sound like we have already done this for all monitored service (when we haven't).

hemajv replied:
updated :+1:

4n4nd commented on README.md:
```diff
@@ -15,21 +15,27 @@ The components we currently have available are:
-TO DO
+* [Prometheus](http://prometheus-portal-opf-monitoring.apps.cnv.massopen.cloud/graph) - Monitoring tool capturing time series metrics for available Operate First services
+* [Grafana](https://grafana-route-opf-monitoring.apps.cnv.massopen.cloud/) - Visualization tool for creating monitoring dashboards
+* [GitHub Receiver](https://github.com/m-lab/alertmanager-github-receiver) - Incident reporting tool for handling outages/incidents
```
Can we add something about alertmanager? Because we can support other types of receivers too

hemajv replied:
good idea! added :+1:

sesheta commented:
New changes are detected. LGTM label has been removed.

sesheta commented:
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/operate-first/SRE/pull/53#pullrequestreview-580750754" title="LGTM">anishasthana</a>*, *<a href="https://github.com/operate-first/SRE/pull/53#pullrequestreview-580764040" title="Approved">HumairAK</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=operate-first%2FSRE).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/operate-first/SRE/blob/master/OWNERS)~~ [HumairAK,anishasthana]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>
<!-- META={"approvers":[]} -->

HumairAK commented:
Bah I didn't notice there was already an approve, woops.

Tessg22 committed:
Clarify when RCA should be ready to be shared publicly

Follow up on Incident Management Process improvement for RCA that has been delayed in public sharing due to unclear responsibilities of the Incident Owner.
https://issues.redhat.com/browse/OHSS-15947

Tessg22 committed:
Remove reference to templates for RCA and PMR docs

mmazur commented on process/incident_management.md:
```diff
@@ -203,7 +203,7 @@ Once our services are recovered and we are back in business then the following s
 * That the above steps are being performed in a timely manner is verified by the Incident Commander.
 
-Templates for RCA and PMR documents can be found here.
+From the incident announcement RCA should be ready to be shared publicly in the next 7 business days. This includes resolving all the outstanding comments and approval if required.
```
```suggestion
The RCA should be ready to be shared publicly within 7 business days of the incident announcement.
This includes ensuring all outstanding comments are resolved and any required approvals granted.
```

Tessg22 committed:
Change wording

Co-authored-by: Mariusz Mazur <mmazur@redhat.com>
